test/CavyTester/src/scenarios/createRef.js

```diff
@@ -0,0 +1,45 @@
+import React, { useRef } from 'react';
+import { Button, TextInput } from 'react-native';
+import { useCavy, hook } from 'cavy';
+
+export const key = 'CreateRef';
+const inputId = `${key}.Input`;
+const buttonId = `${key}.Button`;
+
+class CreateRefTest extends React.Component {
+  textInput = React.createRef();
```

Comment:
For style consistency, please could you import `createRef` in the same way as we do `useRef`?

Reply:
I'd also expect to see this `createRef()` in a constructor for this class component.

Reply:
Well this uses ESnext, no need for a constructor here. But I will refactor and also make sure binding is done manually then. I think it is bad practice to not use modern ESnext functionality fron binding, but that's just my opinion.

Reply:
I will also import Component separately from React then, otherwise this is still a mix and match